frontend/assets/images/inline-svgs/raw/grid-view.svg

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="16px" height="14.4px" viewBox="0 0 16 14.4" enable-background="new 0 0 16 14.4" xml:space="preserve">
+<rect x="-40" y="0" fill="#FFFFFF" width="4" height="2.4"/>
+<rect x="-40" y="4" fill="#FFFFFF" width="4" height="2.4"/>
+<rect x="-40" y="7.999" fill="#FFFFFF" width="4" height="2.4"/>
+<rect x="-40" y="11.999" fill="#FFFFFF" width="4" height="2.4"/>
+<rect x="-34.4" y="0" fill="#FFFFFF" width="10.4" height="2.4"/>
+<rect x="-34.4" y="4" fill="#FFFFFF" width="10.4" height="2.4"/>
+<rect x="-34.4" y="7.999" fill="#FFFFFF" width="10.4" height="2.4"/>
+<rect x="-34.4" y="11.999" fill="#FFFFFF" width="10.4" height="2.4"/>
+<rect fill="#FFFFFF" width="7.2" height="6.4"/>
+<rect y="8" fill="#FFFFFF" width="7.2" height="6.4"/>
+<rect x="8.8" fill="#FFFFFF" width="7.2" height="6.4"/>
+<rect x="8.8" y="8" fill="#FFFFFF" width="7.2" height="6.4"/>
+<g>
+</g>
+<g>
```

Comment:
Should probably clean this up but the output version is cleaned so it's not essential.